src/docker-existing-docker-compose/.devcontainer/docker-compose.yml

```diff
@@ -22,5 +22,5 @@ services:
     #   - seccomp:unconfined
 
```

Comment:
Can you bump patch version in ? Thanks! 

https://github.com/devcontainers/templates/blob/5986da0e79d260ea4141106240927e770350111c/src/docker-existing-docker-compose/devcontainer-template.json#L3